source/04_best_practice/ddp.rst

```diff
@@ -0,0 +1,147 @@
+# Distributed Training with DDP in DI-engine  
+===========================================  
+
+When accelerating the reinforcement learning (RL) training process, **Distributed Data Parallel (DDP)** is an effective approach.  
+This article provides a detailed explanation of how to configure and use DDP training in **DI-engine**, demonstrated through the example `pong_dqn_ddp_config.py`.  
```

Comment:
This article demonstrates how to configure and use DDP training in **DI-engine**, through the example `pong_dqn_ddp_config.py`.